lib/pacemaker/pcmk_sched_colocation.c

```diff
@@ -1262,7 +1265,6 @@ pcmk__add_colocated_node_scores(pe_resource_t *rsc, const char *log_id,
         } else {
             work = pcmk__copy_node_table(rsc->allowed_nodes);
         }
-        clear_node_weights_flags(flags, rsc, pe_weights_init);
```

Comment:
FYI, `clear_node_weights_flags` is no longer used anywhere after this. We can keep it in case it comes in handy later, or scrap it to clean up.

Reply:
Whoops, will drop it